superset/reports/notifications/slack.py

```diff
@@ -184,13 +184,16 @@ def send(self) -> None:
             client = WebClient(token=token, proxy=app.config["SLACK_PROXY"])
             # files_upload returns SlackResponse as we run it in sync mode.
             if files:
-                for file in files:
+                for idx, file in enumerate(files, 1):
```

Comment:
Is there no way of specifying the file extension during the file creation stage?

Reply:
Thank you for review..!

I think it is proper place to define file extension due to the current line 178 on this method.
```
file_type = "csv" if self._content.csv else "png"
```